backend/src/test/java/com/funeat/recipe/application/RecipeServiceTest.java

```diff
@@ -545,6 +549,188 @@ class likeRecipe_실패_테스트 {
         }
     }
 
+    @Nested
+    class writeCommentOfRecipe_성공_테스트 {
+
+        @Test
+        void 꿀조합에_댓글을_작성할_수_있다() {
+            // given
+            final var category = 카테고리_추가_요청(new Category("간편식사", CategoryType.FOOD, "siksa.jpeg"));
+            final var product1 = new Product("불닭볶음면", 1000L, "image.png", "엄청 매운 불닭", category);
+            final var product2 = new Product("참치 삼김", 2000L, "image.png", "담백한 참치마요 삼김", category);
+            final var product3 = new Product("스트링 치즈", 1500L, "image.png", "고소한 치즈", category);
```

Comment:
```suggestion
            final var category = 카테고리_간편식사_생성();
            final var product1 = 상품_삼각김밥_가격1000원_평점2점_생성(category);
            final var product2 = 상품_삼각김밥_가격3000원_평점4점_생성(category);
            final var product3 = 상품_삼각김밥_가격2000원_평점3점_생성(category);
```
이거 다른 테스트들처럼 fixture 사용하는 거 어떤가요??
밑에 Request도!

Reply:
좋습니다!! 감사합니다